website/www/site/content/en/blog/beam-sql-with-notebooks-and-dataflow.md

```diff
@@ -0,0 +1,793 @@
+---
+title:  "Running SQL with Apache Beam Notebooks and Google Cloud Dataflow"
```

Comment:
Should we make this title use Beam terms instead of the GCP products?

Reply:
How about `Running Beam SQL in notebooks`?

Reply:
SGTM!